src/test/java/seedu/address/model/person/PersonTagsContainsTagsPredicateTest.java

```diff
@@ -0,0 +1,101 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.tag.Tag;
+import seedu.address.testutil.PersonBuilder;
+
+public class PersonTagsContainsTagsPredicateTest {
+
+    @Test
+    public void equals() {
+        List<String> firstPredicateTags = Collections.singletonList("first");
+        List<String> secondPredicateTags = Arrays.asList("first", "second");
+        List<Tag> firstPredicateTagList = firstPredicateTags.stream().map(Tag::new).collect(Collectors.toList());
+        List<Tag> secondPredicateTagList = secondPredicateTags.stream().map(Tag::new).collect(Collectors.toList());
+        PersonTagsContainsTagsPredicate firstPredicate = new PersonTagsContainsTagsPredicate(firstPredicateTagList);
+        PersonTagsContainsTagsPredicate secondPredicate = new PersonTagsContainsTagsPredicate(secondPredicateTagList);
+
+        // same object -> returns true
+        assertTrue(firstPredicate.equals(firstPredicate));
+
+        // same values -> returns true
+        PersonTagsContainsTagsPredicate firstPredicateCopy = new PersonTagsContainsTagsPredicate(firstPredicateTagList);
+        assertTrue(firstPredicate.equals(firstPredicateCopy));
+
+        // different types -> returns false
+        assertFalse(firstPredicate.equals(1));
+
+        // null -> returns false
+        assertFalse(firstPredicate.equals(null));
+
+        // different person -> returns false
+        assertFalse(firstPredicate.equals(secondPredicate));
+    }
+
+    @Test
+    public void test_tagsContainTags_returnsTrue() {
```

Comment:
Would breaking test cases out as different methods be clearer?

This one is more of a personal preference but I find it helpful to not group too many assertion tests together